3.0/vector_tile.proto

```diff
@@ -44,6 +44,46 @@ message Tile {
                 // A detailed description on geometry encoding is located in 
                 // section 4.3 of the specification.
                 repeated uint32 geometry = 4 [ packed = true ];
+                
+                // Properties replace existing tags field and
+                // uses the properties field instead. This would only be used if version
+                // for a layer is 3 or greater and tags should not be used at that point
+                //
+                // The properties field is much like the tags value in the it is two integers
+                // pairs that reference key and value pairs however, it is broken out into a
+                // "key_index" and an "complex_value". 
+                // 
+                // The "key_index" is much like the key index in the use for tags, but instead
+                // of pointing to "keys" field in the Layers, it points to the "string_values".
+                // This is the same value store as strings for use in values, so duplicates here
+                // will be pointing to the same indexed position. 
+                //
+                // An complex value has two parts, the first 3 bits are the type bits
+                // and the remaining bits are the parameter bits. What is stored in the parameter
+                // bits is dependant on what the type bit is selected. For example for inline values,
+                // the parameter field is not an index but simply a value. For other types it might
+                // be an index position into a value storage of the layer.
+                // 
+                // uint64t type = complex_value & 0x7; // First 3 Bits
+                // uint64t parameter = complex_value >> 3; 
+                // 
+                //     Type     | Id  | Parameter
+                // ---------------------------------
+                // inline int   |  0  | value of integer ( values between -2^60+1 to 2^60-1 )
```

Comment:
Please use `sint` instead of `int` since this would only be preferred over `uint` when values are negative.